paasta_tools/cli/cli.py

```diff
@@ -84,6 +84,10 @@ def get_argparser():
     )
 
     subparsers = parser.add_subparsers(help="[-h, --help] for subcommand help")
+    subparsers.required = True
+    # https://stackoverflow.com/questions/18282403/argparse-with-required-subcommands/18283730#18283730
+    subparsers.dest = 'command'
```

Comment:
Till we upgrade to a version of python with that PR, then I should ship what I have as is, correct?